extratrees/src/main/java/binnie/extratrees/village/VillageHopeField.java

```diff
@@ -26,10 +26,6 @@
 	private Block cropTypeC;
 	private Block cropTypeD;
 
-	@SuppressWarnings("unused")
-	public VillageHopeField() {
```

Comment:
Is there a reason why this is here? Seems very strange.

Reply:
It is used for the construction of the piece at `MapGenStructureIO#getStructureComponent`. It is needed for the construction of  the piece at the loading of the world like the empty constructor of the TileEntity.